tests/test_alternatives.py

```diff
@@ -1,40 +1,58 @@
 import pytest
-from plute.text.utils.alternatives import flatten_alternatives, serialize_alternatives
+from plute.text.utils.utterances import serialize_utterances, merge_utterances, utterances_to_texts
 
 
-flatten_alternatives_test_cases = [
+flatten_utterances_test_cases = [
     (
-        [[{"transcript": f"text_{i}"}] for i in range(5)],
-        True,
-        [f"text_{i}" for i in range(5)]
-    ),
-    (
-        [[{"transcript": f"text_{i}"}] for i in range(5)],
-        False,
-        [{"transcript": f"text_{i}"} for i in range(5)]
+        [[{"transcript": f"text_{i}_{j}", "confidence": 0} for j in range(3)] for i in range(3)],
+        [[
+            {'confidence': 0, 'transcript': 'text_0_0 text_1_0 text_2_0'},
```

Comment:
inconsisten quotes

Reply:
paste from terminal, btw I ran the linter.